src/membarrier.h

```diff
@@ -50,11 +50,11 @@
 # define IMB asm volatile (".arch armv8.1-a \n\t isb" : : : "memory")
```

Comment:
These indentation fixes were part of a suggested fix for a previous commit with membarrier.h, if I remember.
If so, please use `git rebase -i` to move this commit to after the earlier commit, and then mark this commit as a 'fixup'.